### DISCUSSION TITLE
Is there a way to open an issue? `Application error: a client-side exception has occurred` on vercel with styled-jsx

### DISCUSSION THREAD

**Discussion by xta** [vercel/next.js#66390] [Help]
### Summary

I was commenting on the issue and gh bot already marked as it as completed. The issue is not completed https://github.com/vercel/next.js/issues/66389

@delbaoliveira I am seeing this issue today in `next: 14.2.3` deployed to vercel. 

This issue has been referenced already:

* https://github.com/vercel/next.js/issues/65772
* https://github.com/vercel/next.js/issues/50751
* https://github.com/vercel/next.js/issues/51101 (supposedly resolved)

@leerob says it's fixed in this thread https://github.com/vercel/next.js/issues/51101#issuecomment-1783664272

Thank you!

### Additional information

```js
tl;dr

vercel/styled-jsx is calling `head.insertBefore` when `head` is `undefined` https://github.com/vercel/styled-jsx/blob/main/src/lib/stylesheet.js#L247-L250
```

### Example

https://github.com/vercel/next.js/issues/66389

**Comment by xta** [comment]
closing due to find a workaround in our next app
